functional-samples/tutorial.open-api-reference/sw-api-list.js

```diff
@@ -0,0 +1,106 @@
+const apiList = [
```

Comment:
suggestion: move to a separate JSON file

Reply:
I think we are already giving an example of how to fetch with the extension tips feature. I think, if SW supported [Import assertions](https://chromestatus.com/feature/5765269513306112), I would be more open to moving this into a separate JSON file.

How about I move it into a separate JS file? 🧐😁